src/frontend/screens/Settings/sections/GeneralSettings/index.tsx

```diff
@@ -19,14 +19,15 @@ import {
   Shortcuts,
   TraySettings,
   UseDarkTrayIcon,
-  WinePrefixesBasePath
+  WinePrefixesBasePath,
+  PlaytimeSync
 } from '../../components'
 
 export default function GeneralSettings() {
   const { t } = useTranslation()
 
   return (
-    <>
+    <div>
```

Comment:
is this change needed for any reason?

I think I used a fragment on purpose back then, to not have an extra div

Reply:
This makes it consistent with other screens and fixes some styling issues (don't remember what exactly)

Reply:
Without div
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/62100117/401e481c-1840-4b73-acd5-7aaec82c37c5)

With div
![image](https://github.com/Heroic-Games-Launcher/HeroicGamesLauncher/assets/62100117/8e95e554-bb8e-4e52-9dde-df1f966e5803)

Other sections like Advanced Settings have divs too, so I made it consistent

Reply:
thanks for the explanation